api/app/sequelize/models/transcript.ts

```diff
@@ -0,0 +1,61 @@
+import { Sequelize, Model, ModelStatic, DataTypes } from "sequelize";
+import { Review } from "./review";
+import { User } from "./user";
+
+class Transcript extends Model {
+  public id!: number;
+  public content!: any;
+  public originalContent!: any;
+  public transcriptHash!: string;
+  public pr_url!: string;
+  public status!: "queued" | "not queued" | "requeued";
```

Comment:
we no longer have the review row and field in the db. It can be removed here

Reply:
Which field are you referring to? Is it the `status` field?